src/DefaultBuilder/test/Microsoft.AspNetCore.Tests/WebApplicationTests.cs

```diff
@@ -386,7 +386,7 @@ public void WebApplicationBuilderApplicationNameDefaultsToEntryAssembly()
         [Fact]
         public void WebApplicationBuilderApplicationNameCanBeOverridden()
         {
-            var assemblyName = Assembly.GetEntryAssembly().GetName().Name;
+            var assemblyName = typeof(WebApplicationTests).Assembly.GetName().Name;
```

Comment:
Can we just pick a random name so we don't accidently lose coverage in case we revert behavior or something?

Reply:
The only reason it *could* be random is because we're not loading the assembly.